### PR TITLE
LPS-75483 Change markup to be the same of Clay

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pedroqueiroz94** [comment]
PR manually tested, waiting for Front End guild review.

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/_timeline-item.scss [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+
+		& .timeline-icon {
+			border: 2px solid #6B6C7E;
+			margin-right: 2px;
+		}
 	}
 
-	& .timeline-icon {
-		border: 2px solid #869CAD;
+	button.btn.btn-default.dropdown-toggle.dropdown-toggle-operator.text-uppercase {
```
Comment (interaminense) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/_timeline-item.scss]: Hey @alinedoleron, 

Could this line be less specific?

Comment (clovisdasilvaneto) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/_timeline-item.scss]: @alinedoleron why this hierarchy is really necessary? Can you select the button element through a class? Like: `.liferay-ddm-form-rule-builder-timeline .dropdown-toggle.dropdown-toggle-operator { ... }`

Comment (alinedoleron) [reply dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/_timeline-item.scss]: Thanks @clovisdasilvaneto and @interaminense. I've made the adjustments.

**Comment by interaminense** [comment]
LGTM!

**Comment by clovisdasilvaneto** [comment]
LGTM

**Pull request merged**